README.md

```diff
@@ -4,6 +4,46 @@ Portable Linux (i.e. a generic Linux version) packaging for [ungoogled-chromium]
 
 Portable Linux builds can run on **any Linux distribution** (that regular Chromium supports).
 
+Portable Linux binary can be installed, alternatively:
+
+```
+$ cd ~/Downloads
+$ wget https://github.com/Eloston/ungoogled-chromium-binaries/releases/download/75.0.3770.80-1.2/ungoogled-chromium_75.0.3770.80-1.2_linux.tar.xz
+```
+Check download by comparing md5, sha1 and sha256 sums with those at UC's download webpage.
+
+Unpack it:
+
+```
+$ sudo tar -xvf ungoogled-chromium_75.0.3770.80-1.2_linux.tar.xz -C /opt
+```
+Before you run, you must set up your system so that the browser's sandboxing will work. There are two options:
+* Use the user namespace sandbox. This is the preferred option.
+* Use the SUID sandbox. This should be used on systems that do not have support for user namespaces.
+
+To quickly determine your current sandbox setup:
+1. Launch UC temporarily from a terminal:
+```
+$ /opt/ungoogled-chromium_75.0.3770.80-1.1_linux/chrome
+```
+2. Navigate to:
+*chrome://sandbox/*
+3. Your screen may declare, in part:
+
+_Sandbox Status_
+
+_Layer 1 Sandbox     Namespace_
+
+_PID namespaces      Yes_
+
+In that case, the browser uses a namespace sandbox and can be run by using the untarred `chrome-wrapper` script.
```

Comment:
Also, these instructions are redundant (and make the document longer). Do we need this here if it's already in the README inside the package?

Reply:
Okay.  I had supplied them here in case it was thought that it would be good to give readers an idea of what would be involved after installing compiled binaries on this README webpage, but all proposed passages on sandboxing and the related discussion on launching with either `chrome-wrapper` or `chrome-sandbox` will be removed from that webpage then.